src/TestInfrastructure/Framework/TestLogger.cs

```diff
@@ -113,13 +113,12 @@ public void WriteLine(string messageFormat, params object[] args)
 
         public void LogVerbose(string message, params object[] args)
         {
-            // Note: verbose messages are not stored so we can't write tests against them.
-            // However, they can be logged to test output window to help with checking the output
-            // that is produced.
+            var logText = "[Verbose] " + GetFormattedMessage(string.Format(System.Globalization.CultureInfo.CurrentCulture, message, args));
```

Comment:
Just make the debug log thing a constructor flag, it would fix the existing tests. But then the question is, is this ticket even worth it? Why was it prioritized?

Reply:
it's worth it I believe because we do not test some part of the functionality. We still have lots of places that we do not test verbose logs but at least this is a start.